### PR TITLE
add emoji placeholder and change message box style

### PR DESCRIPTION
**Pull request by Hongzred**
Description
add placeholder for emoji
align top avatar
add border to message box
dynamically resize message box

Testing
ng test & npm start

### CODE REVIEW

**Comment by Hongzred** [comment]
> instead of having a chatbox ui like this
> image
> can you try something like the following instead?
> image

I think add it to the end of message box will be hard to see if user sends long text of message. I think it will be better to stick next to user's avater

File: src/app/chatbox/chatbox.component.scss [hunk 1]
```diff
@@ -96,4 +96,30 @@ section .mat-list .mat-list-avatar{
...
+}
+
+.messages {
+    word-break: break-word;
+    overflow: hidden;
+    display: flex;
+    align-items: flex-start;
+    padding: 3px 3px 0px 3px;
+    margin: 1px;
+    // border: 1px solid;
```
Comment (shiBinH) [inline src/app/chatbox/chatbox.component.scss]: Can just remove commented out code.

File: src/app/chatbox/chatbox.component.html [hunk 2]
```diff
@@ -68,21 +68,27 @@ <h2>
...
           matListAvatar
           matTooltip="{{ event.from === this.userInfo.uid ?  userInfo.displayName : this.selectedConversation.name}}"
           src="https://randomuser.me/api/portraits/thumb/lego/{{ i % 10 }}.jpg"
         />
-        <p *ngIf="event.type == 'text'" [dir]="'ltr'">{{ event.text }}</p>
+        <div>&#9889;</div> <!-- placeholder for emoji-->
+        </div>
+        <p *ngIf="event.type == 'text'" [dir]="'ltr'" class="dialog__box">{{ event.text }}</p>
         <p *ngIf="event.type == 'member:joined'" class="text-center">
           <!-- <b>{{ selectedConversation.members[event.from].name }}</b> -->
```
Comment (shiBinH) [inline src/app/chatbox/chatbox.component.html]: Please remove.

Comment (Hongzred) [reply src/app/chatbox/chatbox.component.html]: removed

File: src/app/chatbox/chatbox.component.scss (hunk 1 above)
Comment (Hongzred) [reply src/app/chatbox/chatbox.component.scss]: removed

**Pull request merged**